protocols/imap/src/test/java/org/apache/james/imap/processor/GetAnnotationProcessorTest.java

```diff
@@ -205,29 +208,100 @@ public void processShouldGetAnnotationsAndReturnCompleteResponseWithoutTruncateD
     }
 
     @Test
-    public void processShouldGetAnnotationsByKeysWithOneDepth() throws Exception {
-        annotationRequest.setKeys(KEYS);
-        annotationRequest.setDepthValue(Depth.ONE);
+    public void processShouldCallGetAnnotationsByKeysWithOneDepth() throws Exception {
+        annotationRequestBuilder.keys(KEYS).depth(Depth.ONE).build();
 
         when(mockStatusResponseFactory.taggedOk(any(String.class), any(ImapCommand.class), any(HumanReadableText.class)))
                 .thenReturn(okResponse);
 
-        processor.process(annotationRequest, mockResponder, mockImapSession);
+        processor.process(annotationRequestBuilder.build(), mockResponder, mockImapSession);
 
         verify(mockMailboxManager, times(1)).getAnnotationsByKeysWithOneDepth(eq(inbox), eq(mailboxSession), eq(KEYS));
     }
 
     @Test
-    public void processShouldGetAnnotationsByKeysWithAllDepth() throws Exception {
-        annotationRequest.setKeys(KEYS);
-        annotationRequest.setDepthValue(Depth.INFINITY);
+    public void processShouldCallGetAnnotationsByKeysWithAllDepth() throws Exception {
```

Comment:
I would say this test is too much based on the implementation.

I think you can remove it...